### PR TITLE
add : image component

### PR DESCRIPTION
**Pull request by Varun-28**
This is my image component which has two types of images : 
1. Responsive image that changes size as per screen size and container in which it is contained
2. Round Image
image1
image2

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
LGTM

File: css/component_styles/image.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* ------- image css ------ */
+
+.img-responsive {
```
Comment (Hamdrive) [inline css/component_styles/image.css:3]: These properties can be applied across all `img`s using `reset.css`
Not mandatory but an idea

**Review by Hamdrive — COMMENTED** [review]
Looks good ready to push

File: css/component_styles/image.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/image.css:3]: ok sure !

**Pull request merged**